app/views/providers/proceeding_loop/substantive_defaults/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_with(
+      model: @form,
+      url: providers_legal_aid_application_substantive_default_path,
+      method: :patch,
+      local: true
+    ) do |form| %>
+  <span class="govuk-caption-l"><%= position_in_array(@proceeding) %></span>
+  <h1 class="govuk-heading-xl"><%= @proceeding.meaning %></h1>
+  <%= page_template template: :basic, form: form do %>
+    <%= form.govuk_radio_buttons_fieldset :accepted_substantive_defaults,
+                                          inline: true,
+                                          legend: { size: 'l', tag: 'h2', text: t('.question') } do %>
+      <p>
+        <strong>Default level of service:</strong> <%= form.object.substantive_level_of_service_name %>
```

Comment:
Should these be translations?

Reply:
Yes, they should be.  Good catch.  Feel free to update if this blocks your subsequent tickets :+1: